src/main/java/ganymedes01/etfuturum/entities/EntityBee.java

```diff
@@ -608,10 +608,6 @@ private boolean isNearTarget() {
 		return this.getBeeFlag(2);
 	}
 
-	private void setNearTarget(boolean p_226452_1_) {
-		this.setBeeFlag(2, p_226452_1_);
-	}
-
```

Comment:
Unused ≠ Dead
Please refrain from removing utility code that may be used in the future.